packages/ffe-form/less/phone-number.less

```diff
@@ -1,5 +1,11 @@
 .ffe-phone-number {
     &__input-group {
+        @media (min-width: @breakpoint-sm) {
+            display: flex;
```

Comment:
Når vi har flex burde vi ha flex-wrap for og klare zoom?

Reply:
Usikker på hva du mener? Flex-wrap er default/nowrap fordi inputene skal ligge ved siden av hverandre. De tåler zoom godt på større skjermer så vidt jeg har kunnet teste meg frem til.